05_syntax_lab/01.py

```diff
@@ -1,5 +1,16 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+
+from random import randint 
+print "Insert Number"
+max = int(raw_input())
+for i in range(9):

```

Comment:
Noticed how the "Insert Number" text is printed and then a newline is inserted after it and before reading? An alternative way to provide prompt text is to pass it as a parameter to raw_input. That way no newline is printed:
`max = int(raw_input("Insert Number:"))`